.github/workflows/ci.yml

```diff
@@ -9,20 +9,30 @@ on:
     branches-ignore: [ staging.tmp ]
 
 jobs:
-  build:
+
+  # Prepare a file containing the commit sha, to be included in the releases
+  prepare:
+    name: Prepare for builds
+    runs-on: ubuntu-18.04
+    steps:
+    - name: Create commit-sha file
+      env:
+        GITHUB_SHA: ${{ github.sha }}
+      run: |
+          echo $GITHUB_SHA > commit-sha
+    - name: Publish commit-sha
+      uses: actions/upload-artifact@v2
+      with:
+        path: commit-sha
+        name: dist
```

Comment:
Unlike wgpu-bin where we have to clone the source repo prior to building, here you can just create the commit-sha file in the publish job and avoid the preparation job completely. It will allow queueing the subsequent jobs earlier

Reply:
I'd assumed the build needed it, but `make package` already does that. We can thus omit this step completely, because the GH release page already reports the commit sha :)